### ISSUE TITLE
修改prometheus.yml时添加错instance，如何删除instance信息

### ISSUE THREAD

**Issue by xue253** [prometheus/prometheus#6631]
## Proposal
**Use case. Why is this important?**

*“Nice to have” is not a good use case. :)*

## Bug Report
**What did you do?**

**What did you expect to see?**

**What did you see instead? Under which circumstances?**

**Environment**

* System information:

	insert output of `uname -srm` here

* Prometheus version:

	insert output of `prometheus --version` here

* Alertmanager version:

	insert output of `alertmanager --version` here (if relevant to the issue)

* Prometheus configuration file:
```
insert configuration here
```

* Alertmanager configuration file:
```
insert configuration here (if relevant to the issue)
```

* Logs:
```
insert Prometheus and Alertmanager logs relevant to the issue here
```

**Comment by brian-brazil** [comment]
It makes more sense to ask questions like this on the [prometheus-users mailing list](https://groups.google.com/forum/#!forum/prometheus-users) rather than in a GitHub issue. On the mailing list, more people are available to potentially respond to your question, and the whole community can benefit from the answers provided.

**Comment by xue253** [comment]
thank you




| |
薛昊宇
|
|
邮箱：xue466088253@163.com
|

签名由 网易邮箱大师 定制

在2020年01月15日 23:20，Brian Brazil 写道：

Closed #6631.
